GenderPayGap.WebUI/Views/Register/ChooseOrganisation.cshtml

```diff
@@ -157,15 +161,19 @@
                                                 else
                                                 {
                                                     <li>
-                                                        <button type="submit" class="link-button" name="command" value="page_@i">@i</button>
+                                                        <button type="submit" class="link-button" name="command" value="page_@i">
+                                                            @i<span class="govuk-visually-hidden">th page of organisations list</span>

```

Comment:
"th" isn't always correct (e.g. 2nd)
Maybe "Page i of organisation list" would be better